wandb/cli/cli.py

```diff
@@ -697,6 +701,11 @@ def _clean():
     default=None,
     help="Set sweep name",
 )
+@click.option(
+    "--project_queue",
```

Comment:
I dislike this being an underscore but I guess we sealed our fate by using `launch_config`. In the launch agent, `--max-jobs` is the param not `--max_jobs`

Reply:
Im going all in on hyphens